### PR TITLE
DOCS-5803-update-confluence-v1.asciidoc

### PR DESCRIPTION
**Pull request by jvancetw**
These are requested suggestions to rewrite the document. DOCS-5803 is the associated Docs ticket.

### CODE REVIEW

File: confluence/confluence-v1.asciidoc [hunk 1]
```diff
@@ -1,77 +1,91 @@
...
-This documentation describes what data the confluence configuration retrieves and the endpoints used.
-This is this list of data that rest connector using confluence configuration will crawl.
+*Confluence Rest Configuration*
+-------------------------------
+This documentation describes aspects of the Confluence Rest Configuration such as the authentication methods, data crawled and retrieved, pagination information, variables used, and endpoints used. Terminology is also provided as a reference.
 
-* spaces 
-* pages (such as Wiki pages), 
-* blog posts,  
-* pages and blog posts comments (indexed as individual solr-docs)
```
Comment (dustinguericke) [inline confluence/confluence-v1.asciidoc:9]: @jvancetw I think this is useful information to have in the introduction. There will be a lot that the connector does NOT crawl, I imagine, so explicitly calling out what it is able to crawl will be important before getting started.

Comment (jvancetw) [reply confluence/confluence-v1.asciidoc:9]: Added.

File: confluence/confluence-v1.asciidoc [hunk 2]
```diff
@@ -1,77 +1,98 @@
-Confluence Rest Configuration
-----------------------------
-This documentation describes what data the confluence configuration retrieves and the endpoints used.
-This is this list of data that rest connector using confluence configuration will crawl.
+*Confluence REST Configuration*
+-------------------------------
```
[the remark this answers is not in the document]
Comment (jvancetw) [reply confluence/confluence-v1.asciidoc:2]: Changed.

File: confluence/confluence-v1.asciidoc [hunk 3]
```diff
@@ -1,77 +1,99 @@
...
-Endpoints to configure with Rest-Connector
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The table below consists of a list of endpoints and descriptions of what each endpoint retrieves.
+* Recrawl that uses the `rely on the strayContentDeletion` parameter
+
+
+*Pagination information*
+------------------------
+
+The `confluence-v1.json` file is also referred to as a recipe. This recipe is configured to use pagination by batch size.
```
Comment (mcondo) [inline confluence/confluence-v1.asciidoc:36]: The `confluence-v1.json` is already referenced in the first section. I think only this is enough -> This recipe is configured to use pagination by batch size.

File: confluence/confluence-v1.asciidoc [hunk 4]
```diff
@@ -1,77 +1,99 @@
...
-* `Root Response Mapping` - Define the mapping between the response and data objects to be indexed.
-* `Parent Data Key` - Key to extract data from root/parent response for subsequent request. The extracted value will be used to replace the ${LW_PARENT_DATA_KEY} variable in the child request configuration (endpoint, query params or body).For example, endpoint: /api/path/${LW_PARENT_DATA_KEY}/additionalInfo
-
-* `Child Data Path` - The path to access a specific object within a child response. For example, to access a list of elements named with key 'objects', the ChildDataPath would be 'objects'.If not provided, the entire response body will be indexed.
+[options="header"]
+|=======================
 
-* `Child Data ID` - The identifier key for the child data object, which value will be the solr-document's ID. Fill this when 'Custom Solr Field' is empty, otherwise the solr-document's ID will be a random UUID.
+|Term|Description
+|`Service Endpoints`|The list of service endpoints where data is retrieved. Each service endpoint configures a root endpoint request.
```
Comment (mcondo) [inline confluence/confluence-v1.asciidoc:86]: The list of service endpoints from which the data is retrieved. Each service endpoint configures a root endpoint request.

File: confluence/confluence-v1.asciidoc [hunk 5]
```diff
@@ -1,77 +1,98 @@
...
-Endpoints to configure with Rest-Connector
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The table below consists of a list of endpoints and descriptions of what each endpoint retrieves.
+* Recrawl that uses the `rely on the strayContentDeletion` parameter
+
+
+*Pagination information*
+------------------------
+
+The `confluence-v1.json` file is also referred to as a recipe and is configured to use pagination by batch size.
```
[the remark this answers is not in the document]
Comment (jvancetw) [reply confluence/confluence-v1.asciidoc]: Changed.

File: confluence/confluence-v1.asciidoc [hunk 6]
```diff
@@ -1,77 +1,98 @@
...
 |=======================
-|Endpoint|Http Verb |Query Param |Description |Request type
-|/wiki/rest/api/content|GET    |type=blogpost&start=0&limit=1&expand=body.storage|get all blogpost from confluence|Data Request
-|/wiki/rest/api/content/${LW_PARENT_DATA_KEY}/child/comment|GET|expand=body.storage|This endpoint gets all blogpost comments. The value of `id` from the main request needs to be assigned to the `${LW_PARENT_DATA_KEY}` variable so that the addtional feature can insert the value when building the GET url.  |Child Endpoint
-|/wiki/rest/api/content | GET |type=paget&start=0&limit=1&expand=body.storage |get all pages from confluence |Data Request
-|/wiki/rest/api/content/${LW_PARENT_DATA_KEY}/child/comment|GET|expand=body.storage|This endpoint gets all page comments. The value of `id` from the main request needs to be assigned to the `${LW_PARENT_DATA_KEY}` variable so that the addtional feature can insert the value when building the GET url.  |Child Endpoint
+|Endpoint|HTTP operation |Query parameter |Description |Request type
+|/wiki/rest/api/content|GET    |type=blogpost&start=0&limit=1&expand=body.storage|Returns all blogposts from the specified URL.|Data Request
```
[the remark this answers is not in the document]
Comment (jvancetw) [reply confluence/confluence-v1.asciidoc]: Changed.

File: confluence/confluence-v1.asciidoc [hunk 7]
```diff
@@ -1,77 +1,98 @@
...
+|/wiki/rest/api/content/${LW_PARENT_DATA_KEY}/child/comment|GET|expand=body.storage|Returns all comments in the blogposts from the specified URL. The value of `id` from the main request needs to be assigned to the `${LW_PARENT_DATA_KEY}` variable so the additional feature can insert that value when building the GET URL.  |Child Endpoint
+|/wiki/rest/api/content | GET |type=paget&start=0&limit=1&expand=body.storage |Returns all pages from the specified URL.|Data Request
```
[the remark this answers is not in the document]
Comment (jvancetw) [reply confluence/confluence-v1.asciidoc]: Changed.

File: confluence/confluence-v1.asciidoc [hunk 8]
```diff
@@ -1,77 +1,98 @@
...
+|/wiki/rest/api/content/${LW_PARENT_DATA_KEY}/child/comment|GET|expand=body.storage|Returns all page comments from the specified URL. The value of `id` from the main request needs to be assigned to the `${LW_PARENT_DATA_KEY}` variable so the additional feature can insert that value when building the GET URL.|Child Endpoint
```
[the remark this answers is not in the document]
Comment (jvancetw) [reply confluence/confluence-v1.asciidoc]: Changed.

File: confluence/confluence-v1.asciidoc [hunk 9]
```diff
@@ -1,77 +1,98 @@
...
-* `Parent Data Key` - Key to extract data from root/parent response for subsequent request. The extracted value will be used to replace the ${LW_PARENT_DATA_KEY} variable in the child request configuration (endpoint, query params or body).For example, endpoint: /api/path/${LW_PARENT_DATA_KEY}/additionalInfo
-
-* `Child Data Path` - The path to access a specific object within a child response. For example, to access a list of elements named with key 'objects', the ChildDataPath would be 'objects'.If not provided, the entire response body will be indexed.
+[options="header"]
+|=======================
 
-* `Child Data ID` - The identifier key for the child data object, which value will be the solr-document's ID. Fill this when 'Custom Solr Field' is empty, otherwise the solr-document's ID will be a random UUID.
+|Term|Description
+|`Service Endpoints`|The list of endpoints (URLs) from which data is retrieved.
+|`Data Request`|A type of request used to retrieve data.
```
[the remark this answers is not in the document]
Comment (jvancetw) [reply confluence/confluence-v1.asciidoc]: Changed.

File: confluence/confluence-v1.asciidoc [hunk 10]
```diff
@@ -1,77 +1,98 @@
...
+|`Child Endpoint`|The type of request to retrieve additional information for the root data objects. It iterates over root data objects, and performs a request for each one.
```
[the remark this answers is not in the document]
Comment (jvancetw) [reply confluence/confluence-v1.asciidoc]: Changed.

File: confluence/confluence-v1.asciidoc [hunk 11]
```diff
@@ -1,77 +1,98 @@
...
-* `Root Response Mapping` - Define the mapping between the response and data objects to be indexed.
-* `Parent Data Key` - Key to extract data from root/parent response for subsequent request. The extracted value will be used to replace the ${LW_PARENT_DATA_KEY} variable in the child request configuration (endpoint, query params or body).For example, endpoint: /api/path/${LW_PARENT_DATA_KEY}/additionalInfo
-
-* `Child Data Path` - The path to access a specific object within a child response. For example, to access a list of elements named with key 'objects', the ChildDataPath would be 'objects'.If not provided, the entire response body will be indexed.
+[options="header"]
+|=======================
 
-* `Child Data ID` - The identifier key for the child data object, which value will be the solr-document's ID. Fill this when 'Custom Solr Field' is empty, otherwise the solr-document's ID will be a random UUID.
+|Term|Description
+|`Service Endpoints`|The list of endpoints (URLs) from which data is retrieved.
```
[the remark this answers is not in the document]
Comment (jvancetw) [reply confluence/confluence-v1.asciidoc]: Changed.

File: confluence/confluence-v1.asciidoc (hunk 3 above)
Comment (jvancetw) [reply confluence/confluence-v1.asciidoc]: Changed.

File: confluence/confluence-v1.asciidoc (hunk 4 above)
Comment (jvancetw) [reply confluence/confluence-v1.asciidoc:86]: Changed.

File: confluence/confluence-v1.asciidoc [hunk 12]
```diff
@@ -1,77 +1,98 @@
...
-|/wiki/rest/api/content | GET |type=paget&start=0&limit=1&expand=body.storage |get all pages from confluence |Data Request
-|/wiki/rest/api/content/${LW_PARENT_DATA_KEY}/child/comment|GET|expand=body.storage|This endpoint gets all page comments. The value of `id` from the main request needs to be assigned to the `${LW_PARENT_DATA_KEY}` variable so that the addtional feature can insert the value when building the GET url.  |Child Endpoint
+|Endpoint|HTTP operation |Query parameter |Description |Request type
+|/wiki/rest/api/content|GET    |type=blogpost&start=0&limit=1&expand=body.storage|Returns all blogposts from the specified URL.|Data Request
+|/wiki/rest/api/content/${LW_PARENT_DATA_KEY}/child/comment|GET|expand=body.storage|Returns all comments in the blogposts from the specified URL. The value of `id` from the main request needs to be assigned to the `${LW_PARENT_DATA_KEY}` variable so the additional feature can insert that value when building the GET URL.  |Child Endpoint
```
[the remark this answers is not in the document]
Comment (jvancetw) [reply confluence/confluence-v1.asciidoc]: Changed.

**Review by mcondo — APPROVED** [review]

**Pull request merged**